src/components/Garden/Feed/AbstainCard.tsx

```diff
@@ -45,17 +45,53 @@ function AbstainCard({ proposal }: AbstainCardProps) {
         `}
       `}
     >
-      <ProposalInfo
-        proposal={proposal}
-        onSelectProposal={handleSelectProposal}
-        loading={false}
-      />
-      <ProposalFooter
+      <AbstainCardHeader
         proposal={proposal}
-        onSelectProposal={handleSelectProposal}
+        handleSelectProposal={handleSelectProposal}
```

Comment:
Here on first sight looks like you changed the name the props, but in fact it's another component :D